libkbfs/tlf_journal.go

```diff
@@ -314,6 +327,26 @@ func makeTLFJournal(
 		bwDelegate:           bwDelegate,
 	}
 
+	if bws == TLFJournalBackgroundWorkPaused {
```

Comment:
might be cleaner to have this function take a `pausedType` and infer `bws` based on it, rather than this way around. But it might not be worth the boilerplate.